src/applications/personalization/profile-2/components/EmailAddressNotification.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+const EmailAddressNotification = ({ signInServiceName }) => {
+  let link;
+  let buttonText;
+
+  if (signInServiceName === 'idme') {
+    link = 'https://wallet.id.me/settings';
+    buttonText = 'ID.me';
+  }
+
+  if (signInServiceName === 'dslogon') {
+    link = 'https://myaccess.dmdc.osd.mil/identitymanagement';
+    buttonText = 'DS Logon';
+  }
+
+  if (signInServiceName === 'mhv') {
+    link = 'https://www.myhealth.va.gov';
+    buttonText = 'My HealtheVet';
+  }
```

Comment:
Not a big difference, but I would do `else if`s here or a `switch` since `signInServiceName` doesn't need to be checked further once it finds a match.